develop/toolkit/parachains/light-clients.md

```diff
@@ -0,0 +1,72 @@
+
+!!!info "Light node or light client?"
+    The terms "light node" and "light client" are interchangeable. Both refer to a blockchain client that syncs without downloading the entire blockchain state. All nodes in a blockchain network are fundamentally clients, engaging in peer-to-peer communication.
+
+Unlike JSON RPC interfaces, where an application must either maintain a list of providers or rely on a single node, light clients are not limited to nor dependent on a single node. They use cryptographic proofs to verify the blockchain's state, ensuring it is both up-to-date and accurate. By verifying only block headers, light clients avoid syncing the entire state, making them ideal for resource-constrained environments.
+
+```mermaid
+flowchart LR
+DAPP([dApp])-- Query Account Info -->LC([Light Client])
+LC -- Request --> FN(((Full Node)))
+LC -- Response --> DAPP
+FN -- Response (validated via Merkle proof) --> LC
+```
+
+In the diagram above, the decentralized application queries on-chain account information through the light client. The light client, which runs as part of the application, requires minimal memory and computational resources. It uses Merkle proofs to verify the state retrieved from a full node in a trust-minimized manner. Polkadot-compatible light clients utilize [warp syncing](https://spec.polkadot.network/sect-lightclient#sect-sync-warp-lightclient){target=\_blank}, which downloads only block headers.
+
+Light clients can quickly verify the blockchain's state, including [GRANDPA finality](/polkadot-protocol/glossary#grandpa){target=\_blank} justifications.
+
+!!!info "What does it mean to be 'trust-minimized'?"
+    "Trust-minimized" means that the light client does not need to fully trust the full node from which it retrieves state. This is achieved through the use of Merkle proofs, which allow the light client to verify the correctness of the state by checking the Merkle tree root.
+
+## JSON RPC and Light Client Comparison
+
+Another common method of communication between a user interface (UI) and a node is through the JSON RPC protocol. Generally, the UI retrieves information from the node, fetches network or [pallet](/polkadot-protocol/glossary#pallet){target=\_blank} data, and interacts with the blockchain. This is typically done in one of two ways:
+
+1. **User-Controlled Nodes** - the UI connects to a node client installed on the user's machine
+    - These nodes are secure, but installation and maintenance can be inconvenient
+2. **Publicly Accessible Nodes** - the UI connects to a third-party-owned publicly accessible node client
+    - These nodes are convenient but centralized and less secure. Applications must maintain a list of backup nodes in case the primary node becomes unavailable
```

Comment:
Are these items supposed to be in sequential order? If not, please switch to unordered list